### DISCUSSION TITLE
Having trouble with formily/antd and React application

### DISCUSSION THREAD

**Discussion by theashishmaurya** [alibaba/formily#3618] [Q&A 2.x]
I am trying to build an application using @formily/antd but while running in React, it has an error saying Uncompiled less.

![image](https://user-images.githubusercontent.com/50199029/206677033-868ff8b5-c484-4195-8c51-12305f12462b.png)

I have no idea how to fix this :/

Can anyone help
here is the link
https://stackblitz.com/edit/vitejs-vite-jswxpr?file=src/App.tsx

**Comment by LiangJunrong** [comment]
+1
